simpleclient_pushgateway/pom.xml

```diff
@@ -62,5 +57,11 @@
            <version>3.9.17</version>
            <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.3.0</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Nitpick: This is not a test dependency, so it would be good to move it up above the comment `<!-- Test Dependencies Follow -->`.

Reply:
I'll make the change.